src/pages/sessions.js

```diff
@@ -1,11 +1,157 @@
 import React from "react";
+import IndexContainer from "../containers/indexContainer";
 import NavigationBar from "../components/Navigationbar";
 import Footer from "../components/Footer";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+import data1 from "../config/sessionToBeConducted.yaml";
```

Comment:
Give better import names, will avoid confusion in the future. Else add comments describing what is data1.

Reply:
@harisumaid name them as the filenames. Always name variables such that another person knows the intent of the variable in one look

Reply:
```suggestion
import data1 from "../config/sessionToBeConducted.yaml";
// data1 contains data about the sessions that are not yet conducted
import data2 from "../config/sessionAlreadyConducted.yaml";
// data2 contains data about the sessions that we have already conducted

```
added those comments

Reply:
I would prefer renaming them.

Reply:
Yes, rename them. Keep the names same as file names. That will be better.